pkgs/applications/misc/qMasterPassword/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, qt5, libX11, libXtst, openssl, libscrypt }:
```

Comment:
https://nixos.org/nixpkgs/manual/#sec-language-qt

> Call your application expression using libsForQt5.callPackage instead of callPackage. Import dependencies unqualified, i.e., qtbase not qt5.qtbase. Do not import a package set such as qt5 or libsForQt5.